python/flask/sample/views/batch_cades_rest.py

```diff
@@ -17,6 +17,13 @@
 from sample.utils import get_rest_pki_client
 from sample.utils import get_security_context_id
 
+# 21-07-2022
+# For some reason, Blueprint has stopped accepting names containing dots('.'),
+# so the exception would be thrown at the blueprint instantiation. In order to 
+# solve that we replaced all occurrences of dots with forward slash ('/'). This fix
+# made the PkiSuiteSamples Flask example run normally again
+__name__ = __name__.replace(".","/")
+
```

Comment:
@rufiss Why the additional line?

Reply:
By further inspecting in the latest [Blueprint documentation](https://flask.palletsprojects.com/en/2.2.x/api/#blueprint-objects), when creating a Blueprint object, the first parameter (`name`) is prepend to the URL endpoint. Therefore, Blueprint no longer allows dots in the name since it would break the URL entirely.
One interesting fact is that somehow this was ignored in previous versions, this can be attributed to how python 2.x and python 3.x interpret `__name__` special variable 

The pic below shows `__name__` value before transformation in `authentication_rest.py`
![image](https://user-images.githubusercontent.com/8876477/186888039-afca4b87-e432-4fcd-8e8f-87c1fc4b72f1.png)

I'm adding these insights to the comments in a future commit